### PR TITLE
Update docker image labels to match updates

### LINKED ISSUE #167: Update labels
Our UBI labels need to be updated, as they still mention SFJ and UBI min.  

Also, the Ubuntu variants are missing some of the labels such as `version`.  We should make them consistent with the labels in the UBI variants.

### PR DESCRIPTION
**Pull request by edavidj**
* Add labels to Ubuntu images that were missing before

* Fix existing labels in UBI images to match the current base images

Fixes #167

### CODE REVIEW

File: releases/20.0.0.3/kernel/Dockerfile.ubi.ibmjava8 [hunk 1]
```diff
@@ -14,8 +14,8 @@ LABEL org.opencontainers.image.authors="Arthur De Magalhaes, Chris Potter" \
       vendor="Open Liberty" \
       name="Open Liberty" \
       version="$LIBERTY_VERSION" \
-      summary="Image for Open Liberty with IBM's SFJ and UBI minimal" \
-      description="This image contains the Open Liberty runtime with IBM's SFJ and Red Hat UBI minimal as the base OS.  For more information on this image please see https://github.com/OpenLiberty/ci.docker#building-an-application-image"
+      summary="Image for Open Liberty with IBM's SFJ and UBI 8" \
```
Comment (arthurdm) [inline releases/20.0.0.3/kernel/Dockerfile.ubi.ibmjava8:17]: This line and the one below still say SFJ.

File: releases/latest/kernel/Dockerfile.ubi.ibmjava8 [hunk 2]
```diff
@@ -14,8 +14,8 @@ LABEL org.opencontainers.image.authors="Arthur De Magalhaes, Chris Potter" \
       vendor="Open Liberty" \
       name="Open Liberty" \
       version="$LIBERTY_VERSION" \
-      summary="Image for Open Liberty with IBM's SFJ and UBI minimal" \
-      description="This image contains the Open Liberty runtime with IBM's SFJ and Red Hat UBI minimal as the base OS.  For more information on this image please see https://github.com/OpenLiberty/ci.docker#building-an-application-image"
+      summary="Image for Open Liberty with IBM's SFJ and UBI 8" \
```
Comment (arthurdm) [inline releases/latest/kernel/Dockerfile.ubi.ibmjava8:17]: change SFJ to `Java` (line below too)

**Comment by arthurdm** [comment]
thanks for getting to the PR so quickly @edavidj!  One general search / replace would be to change `OpenJ9` to `AdoptOpenJDK with OpenJ9`.

File: releases/20.0.0.3/kernel/Dockerfile.ubi.ibmjava8 (hunk 1 above)
Comment (edavidj) [reply releases/20.0.0.3/kernel/Dockerfile.ubi.ibmjava8:17]: Ah alright, wasn't sure if that needed updating or not. I'll add those changes in 👍

**Review by arthurdm — APPROVED** [review]
awesome - thanks @edavidj!

**Pull request merged**